docs/kms/README.md

```diff
@@ -108,6 +108,15 @@ export MINIO_SSE_VAULT_NAMESPACE=ns1
 
 Note: If [Vault Namespaces](https://learn.hashicorp.com/vault/operations/namespaces) are in use, MINIO_SSE_VAULT_NAMESPACE variable needs to be set before setting approle and transit secrets engine.
 
+Advanced options:
+MINIO_SSE_VAULT_CLIENT_TIMEOUT can be set to a custom timeout, defaults to 60 seconds.
+MINIO_SSE_VAULT_RATE_LIMIT limits the rate at which requests are sent to Vault,and can be specified in the format rate:[burst] where rate and burst are "operations per second". If burst is unspecified, it defaults to rate. 
+```
+export MINIO_SSE_VAULT_CLIENT_TIMEOUT=50  # timeout duration in seconds
+export MINIO_SSE_VAULT_RATE_LIMIT=1000:10 # rate limit to 50 requests, with burst limit of 10
```

Comment:
Is this a typo? Should `MINIO_SSE_VAULT_RATE_LIMIT=1000:10 # rate limit to 50 requests` be `MINIO_SSE_VAULT_RATE_LIMIT=1000:10 # rate limit to 1000 requests`